### PR TITLE
Adding Application layer for Users

### PR DESCRIPTION
**Pull request by cathyAkoth**

### CODE REVIEW

File: Application/Models/Users/AddUserModel.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
```
Comment (okellodaniel) [inline Application/Models/Users/AddUserModel.cs:1]: The User models appear to be empty, please look into this

**Review by okellodaniel — COMMENTED** [review]
Hello Cathy, please re-push code to this branch

File: Application/Models/Users/AddUserModel.cs (hunk 1 above)
Comment (cathyAkoth) [reply Application/Models/Users/AddUserModel.cs:1]: Check out Ticket-Application-Feature3.  Work appears

**Comment by cathyAkoth** [comment]
closing this pull request

**Pull request closed without merging**